### ISSUE TITLE
Missing icons into Geary

### ISSUE THREAD

**Issue by Jupiter007-43** [ubuntu/yaru#2238]
Some icons into Geary are missing:

![Capture d’écran du 2020-07-09 13-24-37](https://user-images.githubusercontent.com/36476595/87034069-8e772300-c1e7-11ea-9de5-85fc49870b0d.png)

Complete list here: https://gitlab.gnome.org/GNOME/geary/-/tree/mainline/icons
There are many so I can help if you want.

**Comment by Feichtmeier** [comment]
Generally a good idea, but be aware that we are entering "danger zone" as soon as we change app icons and symbolic icons of "third party gnome apps"

**Comment by Jupiter007-43** [comment]
Yes, and this is as far as I'll go. But some of this icons just need a link to the yaru ones (like tags).
But if you don't want to change these icons I don't mind at all. 🤷

**Comment by Jupiter007-43** [comment]
I think it's actually better to do nothing at all.

**Comment by Feichtmeier** [comment]
If they use generic icon names that we should be on feature parity. But if they use specific icon names like "org.geary.Inbox.svg" or whatever, then we should not include those. I hope you understand what I mean :thinking:

**Comment by Jupiter007-43** [comment]
None of them are using the org.geary.xxx name pattern. For example inbox icon is named mail-inbox-symbolic.svg I think it is that you mean with with generic name.
🤔 So we can include them?

> I hope you understand what I mean

I completely understand. 😉

**Comment by Feichtmeier** [comment]
Yes, if they are generic, we can include them.

**Comment by Jupiter007-43** [comment]
Ok, so I reopen the issue.
